README.md
# Handreiking basisfactuur Rijk

Deze openbare repository is bedoeld om in samenwerkingsverband de _Handreiking basisfactuur Rijk_ te onderhouden en te publiceren.

De volgende ingangen zijn hierbij relevant:

- [Handreiking basisfactuur Rijk (onderhoudsversie)](https://logius-standaarden.github.io/basisfactuur-rijk/)
- [Handreiking basisfactuur Rijk (statische versie)](https://logius-standaarden.github.io/basisfactuur-rijk/snapshot.html)
- [Handreiking basisfactuur Rijk (gepubliceerde versie)]() &middot; <small>_(werkende link volgt ná daadwerkelijke 1<sup>e</sup> publicatie)_</small>

CHANGELOG.md
# Logboek van wijzigingen voor de basisfactuur Rijk

> _Het format van dit wijzigingbestand is gebaseerd op [Keep a Changelog](https://keepachangelog.com/nl/1.1.0/) en maakt gebruik van [Semantic Versioning](https://semver.org/spec/v2.0.0.html)_.

> _Inhoudelijke wijzigingen van de basisfactuur Rijk worden voorafgegaan door (BFR)._


## [v1.0.9] - 2023-11-12

### Fixed

- (BFR) Issue #22 opgelost door BR-NL-BFR-3 niet te testen middels `string-length`, maar middels `count` in [Schematron BFR](./technische-documentatie/basisfactuur-rijk.sch) (v0.10.0 → v0.10.1).

## [v1.0.8] - 2023-08-29

### Added

- (BFR) In alle `assert` elementen de attributen `id` (ten behoeve van het melding-ID) en `flag` (ten behoeve van het severity level) toegevoegd in [Schematron BFR](./technische-documentatie/basisfactuur-rijk.sch) (v0.9.2 → v0.10.0).


## [v1.0.7] - 2023-08-18

### Added

- (BFR) Ontbrekend element 7.3 (Naam/omschrijving van de geleverde goederen of diensten) in [Overzicht verplichte en optionele elementen voor de basisfactuur Rijk](./overzicht-elementen-basisfactuur-rijk.md) toegevoegd.

### Changed

- (BFR) Meldingsteksten [schematron](./technische-documentatie/basisfactuur-rijk.sch) aangepast vanwege reviews (v0.9.1 → v0.9.2)
- (BFR) Typfouten in [Overzicht verplichte en optionele elementen voor de basisfactuur Rijk](./overzicht-elementen-basisfactuur-rijk.md) hersteld.
- (BFR) Afkortingen EG en TC omgezet naar de volledige termen in [de configuratie](/js/config.js).
- Logboek van wijzigingen voor schematron samengevoegd met deze [CHANGELOG.md](./CHANGELOG.md). Relevante links aangepast waar nodig.
- Overbodige prefix `ep` uit de repositorynaam gehaald. Relevante links aangepast waar nodig.


## [v1.0.6] - 2023-06-18

### Added

- (BFR) Technische documentatie toegevoegd - voor de inhoud, zie [technische-documentatie/README.md](./technische-documentatie/README.md).
- Dit logboek van wijzigingen toegevoegd ([CHANGELOG.md](./CHANGELOG.md)).
- Algemene [.gitattributes](./.gitattributes) toegevoegd ter ondersteuning van het bepalen van de bestanden voor de technische release.
- Algemene [.gitignore](./.gitignore) toegevoegd.

### Changed

- (BFR) Paragraaf Hulp bij e-factuur sturen naar de Rijksoverheid aangepast.
- (BFR) Verwijzing naar de technische documentatie aangepast naar de [laatste release](https://github.com/Logius-standaarden/ep-basisfactuur-rijk/releases/latest) van de publicatie;
- Directory-structuur aangepast om aan te sluiten bij de nieuw toegevoegde technische documentatie.
- Logboek van wijzigingen voor schematron aangepast aan het format van [Keep a Changelog](https://keepachangelog.com/nl/1.1.0/).
- Cosmetische aanpassingen van de document-koptekst.


## [v1.0.5] - 2023-05-23

### Added
- Logboek van wijzigingen voor schematron toegevoegd.


## [v1.0.4] - 2023-05-15

### Changed

- (BFR) [Schematron](./technische-documentatie/basisfactuur-rijk.sch) geactualiseerd naar versie v0.9.1:
  - [v0.8.0] - 2023-04-11
    - Vereenvoudiging tests naar aanleiding van extra tests eerder in de keten.
  - [v0.8.1] - 2023-04-13
    - Validatie van optioneel element doc:Invoice/cbc:UBLVersionID verwijderd.
    - Ontbrekende validatieregel BR-NL-BFR-30 toegevoegd.
  - [v0.9.0] - 2023-04-14
    - Volgnummeraanpassingen zodat er nu 4 opeenvolgende meldingsnummers zijn in plaats van op testcase gebaseerde meldingsnummers (19 → 1; 22 → 2; 30 → 3; 37 → 4).
    - Meldingen zijn iets generieker gemaakt.
  - [v0.9.1] - 2023-05-15
    - Alle contexten absoluut gemaakt.
    - Bij elke validatieregel (BR) kort opgenomen wat de inhoud van de regel behelst. Inclusief de naam van test-xml.
    - BR-NL-BFR-1 verplaatst pattern id="type-restrictions" naar id="cardinality-redefines".
    - BR-NL-BFR-3 test vereenvoudigd.
    - BR-NL-BFR-2 en BR-NL-BFR-4 context vereenvoudigd én extra informatie over de gebruikte waarden opgenomen in de melding.


## [v1.0.3] - 2023-03-21

### Changed

- Cosmetische aanpassingen van de document-koptekst.


## [v1.0.2] - 2023-03-12

### Added

- (BFR) Schematron validatiebestand ([basisfactuur-rijk.sch](./technische-documentatie/basisfactuur-rijk.sch)) toegevoegd (v0.7.0).


## [v1.0.1] - 2023-02-14

### Added

- (BFR) Eerste gereviewde werkversie van de nieuwe publicatievorm voor de <em>Handreiking basisfactuur Rijk</em>, gebaseerd op de eerder gepubliceerde versie van december 2022.

## Oudere versies zonder wijzigingenlogs

* **[v1.0.0-7] - 2022-12-14**
* **[v1.0.0-6] - 2021-05-28**
* **[v1.0.0-5] - 2020-12-22**
* **[v1.0.0-4] - 2020-10-07**
* **[v1.0.0-3] - 2020-07-31**
* **[v1.0.0-2] - 2020-05-07**
* **[v1.0.0-1] - 2018-01-30**

js/config.js
var respecConfig = {
  useLogo: true,
  useLabel: true,
  license: "cc0",
  //voor specStatus mogelijkheden zie https://github.com/Logius-standaarden/respec/wiki/specStatus
  specStatus: "WV",
  //voor specType mogelijkheden zie https://github.com/Logius-standaarden/respec/wiki/specType
  specType: "HR",
  pubDomain: "ep",
  shortName: "basisfactuur-rijk",
  publishDate: "2023-11-12",
  publishVersion: "v1.0.9",
  title: "Handreiking basisfactuur Rijk",
  previousPublishVersion: "v1.0.8",
  previousPublishDate: "2023-08-29",
  previousMaturity: "WV",
  content:
    {
      "handreiking-basisfactuur-rijk": "",
      "overzicht-elementen-basisfactuur-rijk": "",
    },
  editors:
    [
      {
        name: "MB",
        company: "Logius",
        companyURL: "https://www.logius.nl",
      },
      {
        name: "MvB",
        company: "Logius",
        companyURL: "https://www.logius.nl",
      },
    ],
  authors:
    [
      {
        name: "Logius Standaarden - team e-procurement",
        company: "Logius",
        companyURL: "https://logius.nl/diensten/e-procurement",
      }
    ],
  github: "https://github.com/Logius-standaarden/basisfactuur-rijk",

  // overwrite sotdText from organisationConfig
  sotdText: {
    nl: {
      sotd: "Status van dit document",
      def: "Dit is de definitieve versie van dit document. Wijzigingen naar aanleiding van consultaties zijn doorgevoerd.",
      wv: `Dit is een werkversie die op elk moment kan worden gewijzigd, verwijderd of vervangen door andere documenten. Het is geen door de Expertgroep eFacturatie goedgekeurde consultatieversie.`,
      cv: `Dit is een door de Expertgroep eFacturatie goedgekeurde consultatieversie. Commentaar over dit document kan gestuurd worden naar `,
      vv: `Dit is een definitief concept van de nieuwe versie van dit document. Wijzigingen naar aanleiding van consultaties zijn doorgevoerd.`,
      basis: "Dit is een document zonder offici&euml;le status.",
    },
    en: {
      sotd: "Status of This Document",
      def: `This is the definitive version of this document. Edits resulting from consultations have been applied.`,
      wv: `This is a draft that could be altered, removed or replaced by other documents. It is not a recommendation approved by the Technical Committee eFacturatie.`,
      cv: `This is a proposed recommendation approved by the Technical Committee eFacturatie. Comments regarding this document may be sent to `,
      vv: `This is the definitive concept of this document. Edits resulting from consultations have been applied.`,
      basis: "This document has no official standing.",
    },
  },

  // Controls if linked "§" section markers are added to a document
  addSectionLinks: true,

  // this parameter will add the tag_name of the latest release to the document Title
  // only set this parameter when a release has been set
  nl_addReleaseTagTitle: false,

  // if nl_markdownEmbedImageInFigure is set to true images in markdown generated content will be surrounded with <figures> element
  // so that figures can be linked are be a part of table of figures
  nl_markdownEmbedImageInFigure: true,

  // Create PDF and link to file in header:
  alternateFormats: [
    {
      label: "pdf",
      uri: "handreiking-basisfactuur-rijk-v1.0.6.pdf",
    },
  ],
};
